### DISCUSSION TITLE
Variables Queries

### DISCUSSION THREAD

**Discussion by gytis-mackevicius** [grafana/grafana-infinity-datasource#41] [Q&A] [answered]
Hello,

Is it possible to use infinity Web API to generate variables in grafana? Maybe is it possible to use infinity Global Queries for variable generation?

Kind regards
Gytis

**Comment by yesoreyeram** [comment, marked as answer]
Currently it is not possible. Feature is already developed. Fixing some bugs. If everything goes well, will release in a week or two.

**Reply by yesoreyeram** [reply]
To give more context #32 needs to be fixed before the release.

**Reply by palmarisiva** [reply]
Is this feature available now?

**Reply by palmarisiva** [reply]
I have got it. Thanks.
